biolink-model.yaml

```diff
@@ -2153,6 +2188,97 @@ slots:
     mappings:
       - rdf:type
 
+  ## Additional Properties for Information Content Entity and Publication Nodes
+
+  doi:
```

Comment:
we typically don't have prefix-specific properties for identifier mappings

Reply:
I was thinking that the DOI  is a somewhat high profile globally relevant identifier namespace for digital  information which is worth enforcing (visibly highlighting) at the Publication node property level.

Reply:
> 
> 
> we typically don't have prefix-specific properties for identifier mappings

OK @cmungall, I've removed the 'doi' as node property and simply added a fresh 'id_prefix' assertion (did it on `InformationContentEntity` since the 'digital object' tagging scope of DOI's is slightly broader than for vanilla publications)